docs/Chip.md

```diff
@@ -0,0 +1,33 @@
+# Grade
+
+The Grade component is a customizable [chip](https://callstack.github.io/react-native-paper/docs/components/Chip/#selectedcolor)-based component that makes the chip component in alignment to BrightlayerUI. Chips are compact elements that can represent inputs, attributes, or actions. They can have an icon or avatar on the left, and a close button icon on the right.
```

Comment:
Add link https://callstack.github.io/react-native-paper/docs/components/Chip/ instead of https://callstack.github.io/react-native-paper/docs/components/Chip/#selectedcolor